src/core/apim_payment_manager.tf

```diff
@@ -830,7 +830,7 @@ module "apim_pm_per_nodo_v1" {
   name                  = "${local.project}-pm-per-nodo-api"
   api_management_name   = module.apim.name
   resource_group_name   = azurerm_resource_group.rg_api.name
-  product_ids           = [module.apim_payment_manager_product.product_id]
+  product_ids           = [module.apim_payment_manager_product.product_id, local.apim_x_node_product_id]
```

Comment:
this is API exposed by the PM, is it correct that it is also of the Nodo?

Reply:
The goal of the PR is to have all API called by Nodo under only one product in order to manage an unique subscription-key for APIs behind APIM:
- https://pagopa.atlassian.net/wiki/spaces/IQCGJ/pages/665485538/RFC+-+Nuova+Connettivit+verso+prodotti+interni
- https://pagopa.atlassian.net/wiki/spaces/PAG/pages/712802680/WIP+A.T.+Nuova+connettivit+aggiunta+subkey